PERFORMANCE.md

```diff
@@ -0,0 +1,127 @@
+
+#### Put Tensor - send image to database
+![Throughput Std Put](/figures/put_tensor_thro_std.png "Throughput Standard")
+
+## Throughput Colocated
+The following are scaling results for a colocated throughput test, run with 48 client threads on each node. The databases were run on 8 CPUs and 8 threads per queue. 
+
+#### Unpack Tensor - retrieve the data
+![Throughput colo Unpack](/figures/unpack_tensor_thro_colo.png "Colocated Throughput")
+
+#### Put Tensor - send image to database
+![Throughput colo put](/figures/put_tensor_thro_colo.png "Colocated Throughput")
+
+## Throughput Performance Analysis
+
+1. The Throughput Standard test density decreases as the database nodes increase. Outliers become greater as client total increases. This is because as there are more clients, there is a longer wait time. We can also tell that the standard takes longer than colocated because the client nodes are having to travel offsight to the database client node. Each database is having to handle more requests since they do not 
+have their own clients. 
+
+2. Put/Unpack tensor for Throughput Colocated stays consistent with density across all client totals.
+It takes much less time to complete using a colocated orchestrator since all the computations are happening on the same node. Less time it takes to communicate since the db and app are on the same node.
+
+## Data Aggregation Standard
+
+The following are scaling results for a data aggregation test, run with 32 client threads on each node. The databases were run on 36 CPUs and 4 threads per dataset. 
+
+#### Get List - retrieve the data from the aggregation list
+![Data Agg Get List](/figures/get_list_data_agg.png "Data Aggregation Standard")
+
+## Data Aggregation Standard Py
+The following are scaling results for a data aggregation py test, run with 32 client threads on each node. The databases were run on 36 CPUs and 4 threads per dataset.
+
+#### Get List - retrieve the data from the aggregation list
+![Data Agg Py Get List](/figures/get_list_data_agg_py.png "Data Aggregation Py Standard")
+
+#### Poll List - check when the next aggregation list is ready
+![Data Agg Py Poll List](/figures/poll_list_data_agg_py.png "Data Aggregation Py Standard")
+
+## Data Aggregation Standard Py Fs
+The following are scaling results for a data aggregation py fs test, run with 32 client threads on each node.
+
+#### Get List - retrieve the data from the aggregation list
+![Data Agg Py Fs Get List](/figures/get_list_data_agg_fs.png "Data Aggregation Py Fs Standard")
+
+#### Poll List - check when the next aggregation list is ready
+![Data Agg Py Fs Poll List](/figures/poll_list_data_agg_fs.png "Data Aggregation Py Fs Standard")
+
+## Data Aggregation Performance Analysis
+
+1. The Standard Data Aggregation test performs the fastest get_list() client function. We
+can assume this is due to the fact that there is one less communication layer
```

Comment:
Do you mean software layer instead of communication layer?

Also, what do you man that the 'Standard Data Aggregation' test performs the fastest?